integrations/pagerDuty.js

```diff
@@ -212,11 +227,11 @@ async function updateIncident(authData, user) {
     statuses: ['triggered', 'acknowledged'],
   }
 
-  sdk.log('')
+  await ux.print('')
```

Comment:
replace this by putting a newline at the end of the previous prompt. I think it makes it show up better on slack, same for the other print below, I think you can safely put that in the spinner.stop